plugins/derlord/OSS.md

```diff
@@ -36,71 +41,66 @@ So feel free to do whatever you want!
 - Drop server/client files into athena-oss -> src/core/client-plugins, client stuff here -> src/core/server-plugins -> server stuff here.
 
 ```typescript
-Imports (Client) ->
-import './OSS/view';
-import './OSS/src/client-events';
+- Imports (Client) ->
+- import './athena-oss/view';
+- import './athena-oss/src/client-events';
 
 Imports (Server) ->
-import './OSS/index';
-
-// src-webviews/pages/components.ts ->
-import ShopUI from './shopUI/ShopUI.vue';
-ShopUI: shallowRef(ShopUI)
-```
-# Setup (Buying Shop - 24/7)
-```typescript
-export const BUYERS: alt.Vector3[] = [
-    { x: 25.980966567993164, y: -1345.6417236328125, z: 28.497024536132812 } as alt.Vector3, // Index 0
-];
-
-// Used to give all the shops are different blip, just copy the line as often as you need it.
-export const buyerBlips = [
-    { sprite: 59, color: 2, text: '24/7 Shop', scale: 1 }, // Index 0
-];
-
-// In OSS each Shop can hold individual items or you can add the same list to a few more stores over and over again.
-// Default List for all 24/7 & LTD Stores.
-const shopList = [
-    { name: 'Bread', dbName: 'bread', price: 75, image: 'crate' },
-    { name: 'Hotdog', dbName: 'Shophotdog', price: 375, image: 'crate' },
-    { name: 'Waterbottle', dbName: 'Shopwater', price: 250, image: 'crate' },
-    { name: 'Cola', dbName: 'Shopcola', price: 250, image: 'crate'},
-    { name: 'Energy Drink', dbName: 'Shopenergy', price: 300, image: 'crate'}
-];
+- import './OSS/index';
 
-export const buyLists = [
-    shopList,
-]; // ADD YOUR LISTS HERE!
+src-webviews/pages/components.ts ->
+- import ShopUI from './shopUI/ShopUI.vue';
+- ShopUI: shallowRef(ShopUI)
 ```
-
-# Setup (Drugsellers)
+# Setup (Shops / TYPE )
 ```typescript
-import * as alt from 'alt-server';
-
-// Shops are based on the index of arrays so -> First index -> First sellList
-export const SELLERS: alt.Vector3[] = [
-    { x: 1163.400634765625, y: -323.938232421875, z: 68.20509338378906 } as alt.Vector3 // SELLER (SHOP-POS) INDEX -> 0
-    // { Add stuff yourself } Just another seller, INDEX 1 -> The secoond list will get into this position.
-];
-
-// Used to give all the shops are different blip, just copy the line as often as you need it.
-export const sellersBlips = [
-    { sprite: 52, color: 1, text: 'Seller Example - Index 0', scale: 1 }, // Index 0
+export const coreShopLocations: IShopLocation[] = [
```

Comment:
Shouldn't we also define the types themselves?

Reply:
Unsure what this exactly means? 🤔